### PR TITLE
[AmHy] [RoRo] [KeEn] Blacklist updates

### PR DESCRIPTION
**Pull request by Vesihiisi**
Update the P31 blacklist based on:

* https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/am_(hy)/matches
* https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/ro_(ro)/matches
* https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/ke_(en)/matches -- technically, no items from this one

### CODE REVIEW

**Comment by lokal-profil** [comment]
I'm not sure about Q123705/neighbourhood. I think this is very often used for a chunk of a city in a particular style which could very well be protected.

**Comment by lokal-profil** [comment]
Maybe add Q233324/seminary

File: importer/mappings/P31_blacklist.json [hunk 1]
```diff
@@ -106,5 +106,33 @@
...
+    {
+        "item": "Q2061186",
+        "comment": "religious order"
+    },
+    {
+        "item": "Q43229",
+        "comment": "organization"
+    },
+    {
+        "item": "Q123705",
```
Comment (lokal-profil) [inline importer/mappings/P31_blacklist.json]: I'm not sure about Q123705/neighbourhood. I think this is very often used for a chunk of a city in a particular style which could very well be protected.

File: importer/mappings/P31_blacklist.json [hunk 2]
```diff
@@ -106,5 +106,33 @@
...
+        "comment": "Berd"
+    },
+    {
+        "item": "Q4671277",
+        "comment": "academic institution"
+    },
+    {
+        "item": "Q414147",
+        "comment": "academy of sciences"
     }
```
Comment (lokal-profil) [inline importer/mappings/P31_blacklist.json]: Maybe add Q233324/seminary

Comment (Vesihiisi) [reply importer/mappings/P31_blacklist.json]: :white_check_mark:

File: importer/mappings/P31_blacklist.json (hunk 1 above)
Comment (Vesihiisi) [reply importer/mappings/P31_blacklist.json]: :white_check_mark:

**Pull request merged**